src/contracts/core/StrategyManager.sol

```diff
@@ -157,7 +199,7 @@ contract StrategyManager is
     ) external nonReentrant {
         (, uint256 sharesToBurn) = EnumerableMap.tryGet(burnableShares, address(strategy));
         EnumerableMap.remove(burnableShares, address(strategy));
-        emit BurnableSharesDecreased(strategy, sharesToBurn);
+        emit BurnOrRedistributableSharesDecreased(OperatorSet(address(this), 0), 0, strategy, sharesToBurn);
```

Comment:
Can we clarify that `burnShares` (and its underlying `burnableShares` mapping) is basically being deprecated here?

It's probably worth changing the storage name to denote that it's deprecated, and running a script to clear the mapping just after the release. That way we can delete this method in a future release.